client/src/components/ItemList.jsx

```diff
@@ -73,9 +87,11 @@ const ItemList = ({ match }) => {
             </p>
           )}
         </div>
-        <LinkButton primary userId={userId}>
-          Add Another Item
-        </LinkButton>
+        {config.token != null && config.onSuccess != null && (
```

Comment:
could you explain when the token would be null but the onSuccess would exist? Just so i can understand

Reply:
oh, no, you are right.  if there is a token, there would most certainly be an onSuccess function.